.gitignore

```diff
@@ -1,3 +1,6 @@
 target
 Cargo.lock
 .dfx
+.vscode/tasks.json
+.vscode/launch.json
+.vscode/bookmarks.json
```

Comment:
Please add ide related gitignores to your own global gitignore file, we can't add for every ide